misc/docs/source/tutorial/code_snippets/xcorr_detector.rst

```diff
@@ -7,18 +7,25 @@ This code snippets shows how to use the functions
 :func:`~obspy.signal.cross_correlation.similarity_detector`.
 
 In the first example we will determine the origin time of the 2017
-Korean nuclear test, by using a template of another test in 2013. We will
-use only s single channel of the station IC.MDJ.
+North Korean nuclear test, by using a template of another test in 2013. We will
+use only a single channel of the station IC.MDJ.
+
 
 .. plot::
-   :include-source:
+    :context:
+    :include-source:
 
     from obspy import read, UTCDateTime as UTC
     from obspy.signal.cross_correlation import correlate_stream_template, similarity_detector
 
     template = read('https://examples.obspy.org/IC.MDJ.2013.043.mseed')
     template.filter('bandpass', freqmin=0.5, freqmax=2)
     template.plot()
+
+.. plot::
+    :context:
```

Comment:
from the Sphinx docs it's quite unclear how long this context stays active..

Reply:
I get your point. If I use :context: reset in the first code cell that should be OK.